EnglishTranslation-master/QUEST_LV_0300.tsv

```diff
@@ -362,77 +362,77 @@ QUEST_LV_0300_20151001_000361	For Death (1)
 QUEST_LV_0300_20151001_000362	I will destroy the integrator
 QUEST_LV_0300_20151001_000363	I will stop helping
 QUEST_LV_0300_20151001_000364	About the integrator
-QUEST_LV_0300_20151001_000365	The movements of monsters are heard!{nl}Destroy the barrier stones by avoiding the monster attacks!
+QUEST_LV_0300_20151001_000365	You hear the movement of monsters!{nl}Destroy the barrier stones by avoiding the monster attacks!
 QUEST_LV_0300_20151001_000366	For Death (2)
 QUEST_LV_0300_20151001_000367	I will go collect some explosives
 QUEST_LV_0300_20151001_000368	Lighting up the explosive
 QUEST_LV_0300_20151001_000369	The barrier stone has been destroyed due to a great explosion!
 QUEST_LV_0300_20151001_000370	For Death( 3)
-QUEST_LV_0300_20151001_000371	Putting the orb
+QUEST_LV_0300_20151001_000371	Putting the orb in place
 QUEST_LV_0300_20151001_000372	Say Farewell
 QUEST_LV_0300_20151001_000373	Help
 QUEST_LV_0300_20151001_000374	Look at the petrification detector at the entrance of the Coastal Fortress
 QUEST_LV_0300_20151001_000375	Escape from the petrified frosts
-QUEST_LV_0300_20151001_000376	As you tried to look at the detector, the petrified frosts are blowing. Go to a safe place by avoiding petrified frosts!
-QUEST_LV_0300_20151001_000377	You were able to escape safely from petrified frosts due to Kupole Meile. Talk with Kupole Meile.
-QUEST_LV_0300_20151001_000378	In order for Kupole Meile to guide the spirits safely from petrified frosts, check all the detectors that you haven't been in this region.
-QUEST_LV_0300_20151001_000379	Tell the circumstances to Kupole Meile
+QUEST_LV_0300_20151001_000376	As you tried to look at the detector, the petrified frosts started to blow. Go to a safe place by avoiding petrified frosts!
```

Comment:
The petrified frosts blow against you as you tried to check the detector. Go to a safe place to avoid the petrified frosts!